packages/backend/package.json

```diff
@@ -101,7 +102,7 @@
         "slackify-markdown": "^4.4.0",
         "sshpk": "^1.17.0",
         "tempy": "^1.0.1",
-        "tsoa": "^4.1.3",
+        "@tsoa/runtime": "^4.1.3",
```

Comment:
moevd tsoa/cli to devdependencies, which is using `merge` and added here tsoa/runtime 
![image](https://github.com/lightdash/lightdash/assets/1983672/9f7081d7-2747-440a-954f-1d6614114c57)